app/helpers/application_helper.rb

```diff
@@ -1,7 +1,7 @@
 module ApplicationHelper
   def full_title page_title = ""
     base_title = t "layouts.application.base_title"
-    if page_title.empty?
+    if page_title.blank?
```

Comment:
em dùng toán tử 3 ngôi cho những khi check if else ngắn gọn hơn nhé:
page_title.blank? ? base_title : page_title + " | " + base_title